CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
 
+### __[v2.4.4]__ - 12.04.2020
+##### Added
+-Telegram Plugin: Wildcards für fetten, kursiven und unterstrichenen Text hinzugefügt
+[#440](https://github.com/Schrolli91/BOSWatch/pull/440)
+
+
```

Comment:
Hier passt irgendwas nicht - scheinbar bist du vom master aus gegangen?
So sieht der Changelog auf dem Dev aktuell aus - das sind alles derzeit unveröffentlichte Änderungen:
https://github.com/Schrolli91/BOSWatch/blob/develop/CHANGELOG.md
Am besten mal in deinem branch ein `git merge develop` machen.

Anpassungen IMMER auf Basis des dev's machen und auch denn Pull Request gegen den Dev stellen.
Der MASTER ist der aktuelle release Stand